index.bs

```diff
@@ -4971,21 +4972,21 @@ partial interface MLGraphBuilder {
         - <dfn>input</dfn>: an {{MLOperand}}. The input tensor.
         - <dfn>options</dfn>: an optional {{MLReduceOptions}}. The optional parameters of the operation.
 
-    **Returns:** an {{MLOperand}}. The reduced output tensor.
+    **Returns:** an {{MLOperand}}. The reduced output tensor. If the input operand is a scalar, the reduction function is applied to the scalar value and the output is also a scalar.
 </div>
 
-<div class="note">
+<div>
     **Reduction types:**
-        - *L1*: Compute the <a href="https://mathworld.wolfram.com/L1-Norm.html">L1 norm</a> of all the input values along the axes.
-        - *L2*: Compute the <a href="https://mathworld.wolfram.com/L2-Norm.html">L2 norm</a> of all the input values along the axes.
-        - *LogSum*: Compute the log value of the sum of all the input values along the axes.
-        - *LogSumExp*: Compute the log value of the sum of the exponent of all the input values along the axes.
-        - *Max*: Compute the maximum value of all the input values along the axes.
-        - *Mean*: Compute the average value of all the input values along the axes.
-        - *Min*: Compute the minimum value of all the input values along the axes.
-        - *Product*: Compute the product of all the input values along the axes.
-        - *Sum*: Compute the sum of all the input values along the axes.
-        - *SumSquare*: Compute the sum of the square of all the input values along the axes.
+        - *L1*: Compute the <a href="https://mathworld.wolfram.com/L1-Norm.html">L1 norm</a>, the sum of the absolute value of all the input values.
```

Comment:
When we say "all the input values", is it only for "all dimensions are reduced" case?

Reply:
No, so _"input values"_ is not clear, I guess. I felt "along the axes" was unclear (since it can be no axes), so I was wanting "input values" to mean all input values under consideration, as specified by the axes parameter. Up top it says _"The values of the resulting tensor are calculated using the specified reduction function that takes as parameters all the input values across the reduced dimensions."_

I'll make the axes definition more explicit.

We could make it clearer that axes are what

Reply:
Take a look at 7fbceaa - does that help?

Reply:
Yes, that looks much clearer, thanks!